pkg/bpf/map.go

```diff
@@ -106,6 +106,10 @@ type Map struct {
 	path string
 	once sync.Once
 	lock sync.RWMutex
+
+	// NonPersistent is true if the map does not contain persistent data
+	// and should be removed on startup.
```

Comment:
Yes, I thought this was cilium specific and that you were removing maps on startup. If it's generic I'm good.